app/jobs/searchgov_domain_destroyer_job.rb

```diff
@@ -1,11 +1,12 @@
 class SearchgovDomainDestroyerJob < ApplicationJob
   queue_as :searchgov
 
-  def perform(searchgov_domain:)
-    searchgov_domain.searchgov_urls.find_each do |url|
-      url.destroy!
-    end
+  def perform(searchgov_domain)
+    searchgov_domain.searchgov_urls.find_each(&:destroy!)
     searchgov_domain.destroy!
+    Resque::Job.destroy('searchgov',
```

Comment:
Can you add a test for this change?. Something along the lines of:
* Enqueue indexing with a SearchgovDomainIndexerJob job
* Execute SearchgovDomainDestroyerJob
* Validate queue is empty